Servers/using-self-service-vm-import.md

```diff
@@ -1,14 +1,14 @@
 {{{
   "title": "Using Self-Service VM Import",
-  "date": "08-17-2016",
-  "author": "Gavin Lai",
+  "date": "11-25-2019",
+  "author": "Matthew Ordman",
   "attachments": [],
   "contentIsHTML": false
 }}}
 
-**Note**: this feature only supports OVFs exported from VMware environments, and certain operating systems. Virtual appliances and OVFs from other hypervisors should be imported as a service task.
+**Note**: this feature only supports OVFs exported from VMware environments, and [supported operating systems](https://www.ctl.io/knowledge-base/support/supported-operating-systems). Virtual appliances and OVFs from other hypervisors should be imported as a [service task](https://www.ctl.io/centurylink-public-cloud/service-tasks/).
```

Comment:
Links to other KB articles need to be a relative link to the .md path
so this KB link should be `../support/supported-operating-systems.md`